en/embedding.html

```diff
@@ -273,10 +389,277 @@ <h2 id="examples">Examples</h2>
+      <td>101</td>
+    </tr>
+    <tr>
+      <td>transformer-end-sequence-token</td>
+      <td>One</td>
+      <td>The start of sequence token</td>
+      <td>numeric</td>
+      <td>102</td>
+    </tr>
+    <tr>
+      <td>pooling-strategy</td>
+      <td>One</td>
+      <td>How the output vectors of the ONNX model is pooled to obtain a single vector representation. Valid values are <code>mean</code> and <code>cls</code></td>
+      <td>string</td>
+      <td>mean</td>
+    </tr>
+  </tbody>
+</table>
+
+<h2 id="embedder-onnx-reference-config">embedder onnx reference config</h2>
+<p>Vespa uses <a href="https://onnxruntime.ai/">ONNXRuntime</a> to accelerate inference of embedding models. These parameters are valid for both 
+<a href="#bert-embedder">bert embedder</a> and <a href="#huggingface-embedder">huggingface embedder</a></p>
+<table class="table">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Occurrence</th>
+      <th>Description</th>
+      <th>Type</th>
+      <th>Default</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>onnx-execution-mode</td>
+      <td>One</td>
+      <td>Low level ONNX execution model. Valid values are <code>parallel</code> or <code>sequential</code>.
+         See <a href="stateless-model-evaluation.html#onnx-inference-options">ONNX inference options</a>. Only relevant for inference on CPU. 
+         See <a href="https://onnxruntime.ai/docs/performance/tune-performance/threading.html">ONNXruntime documentation</a> on threading.
+        </td>
+      <td>string</td>
+      <td>sequential</td>
+    </tr>
+    <tr>
+      <td>onnx-interop-threads</td>
+      <td>One</td>
+      <td>Low level ONNX setting. See <a href="stateless-model-evaluation.html#onnx-inference-options">ONNX inference options</a>. Only relevant for inference on CPU.</td>
+      <td>numeric</td>
+      <td>1</td>
+    </tr>
+    <tr>
+      <td>onnx-intraop-threads</td>
+      <td>One</td>
+      <td>Low level ONNX setting. See <a href="stateless-model-evaluation.html#onnx-inference-options">ONNX inference options</a>. Only relevant for inference on CPU.</td>
+      <td>numeric</td>
+      <td>4</td>
+    </tr>
+    <tr>
+      <td>onnx-gpu-device</td>
+      <td>One</td>
+      <td>The GPU device to run the model on. See <a href="stateless-model-evaluation.html#onnx-inference-options">ONNX inference options 
+        and <a href="vespa-gpu-container.html">configuring GPU for Vespa container</a>. Use <code>-1</code> to not use GPU for the model, even
+        if the instance has available GPUs.
+      </td>
+      <td>numeric</td>
+      <td>0</td>
+    </tr>
+  </tbody>
+</table>
+
+<h2 id="huggingface-tokenizer-reference-config">huggingface tokenizer reference config</h2>
+<table class="table">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Occurrence</th>
+      <th>Description</th>
+      <th>Type</th>
+      <th>Default</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>model</td>
+      <td>One</td>
```

Comment:
`model` can be specified multiple times for language specific models (in combination with `language` attribute)